packages/components/collapsible-panel/src/collapsible-panel.js

```diff
@@ -38,77 +39,83 @@ const CollapsiblePanel = (props) => {
       isDefaultClosed={props.isDefaultClosed}
     >
       {({ isOpen, toggle, containerStyles, registerContentNode }) => (
-        <div
-          css={[baseContainerStyles, getThemeStyle(props.theme)]}
-          // Allow to override the styles by passing a `className` prop.
-          // Custom styles can also be passed using the `css` prop from emotion.
-          // https://emotion.sh/docs/css-prop#style-precedence
-          className={props.className}
+        <Constraints.Horizontal
+          max={Constraints.parseHorizontalConstraintProp(
+            props.horizontalConstraint
+          )}
```

Comment:
```suggestion
          max={props.horizontalConstraint}
```

The `parseHorizontalConstraintProp` is only used for backwards compatibility. We can simply require the new prop now as-is.